Stellar-operation-manage-asset.x

```diff
@@ -98,37 +123,62 @@ struct ManageAssetOp
+    //: Means that specified action in `data` of ManageSignerOp was successfully executed
     SUCCESS = 0,                       // request was successfully created/updated/canceled
 
     // codes considered as "failure" for the operation
-	REQUEST_NOT_FOUND = -1,           // failed to find asset request with such id
-    INVALID_SIGNATURE = -2,           // only asset pre issuer can change asset pre issuer
-	ASSET_ALREADY_EXISTS = -3,	      // asset with such code already exist
+    //: There is no `CREATE_ASSET` or `UPDATE_ASSET` request with such id
+    REQUEST_NOT_FOUND = -1,           // failed to find asset request with such id
+    //: only asset pre issuer can change asset pre issuer
+    INVALID_SIGNATURE = -2,
+    //: Not allowed to create asset with code which already used for an other asset
+    ASSET_ALREADY_EXISTS = -3,	      // asset with such code already exist
+    //: Not allowed to set max issuance amount that
+    //: less than sum of issued and pendingIssuance and available for issuance
     INVALID_MAX_ISSUANCE_AMOUNT = -4, // max issuance amount is 0
-	INVALID_CODE = -5,                // asset code is invalid (empty or contains space)
+    //: Not allowed use asset code which is empty or contains space
+    INVALID_CODE = -5,                // asset code is invalid (empty or contains space)
+    //: Not allowed to set pre issuer equals to exiting pre issuer
     INVALID_PRE_ISSUER = -6,          // pre issuer is the same as existing
-	INVALID_POLICIES = -7,            // asset policies (has flag which does not belong to AssetPolicies enum)
-	ASSET_NOT_FOUND = -8,             // asset does not exists
-	REQUEST_ALREADY_EXISTS = -9,      // request for creation of unique entry already exists
-	STATS_ASSET_ALREADY_EXISTS = -10, // statistics quote asset already exists
-	INITIAL_PREISSUED_EXCEEDS_MAX_ISSUANCE = -11, // initial pre issued amount exceeds max issuance amount
+    //: Not allowed to set policies which is not declared
+    INVALID_POLICIES = -7,            // asset policies (has flag which does not belong to AssetPolicies enum)
+    //: There is no asset with such code
+    ASSET_NOT_FOUND = -8,             // asset does not exists
+    //: Request for such asset already exists
+    REQUEST_ALREADY_EXISTS = -9,      // request for creation of unique entry already exists
+    //: Not allowed to create two or more assets with `STATS_QUOTE_ASSET` policy
+    STATS_ASSET_ALREADY_EXISTS = -10, // statistics quote asset already exists
+    //: Not allowed to set pre issued amount that greater than max issuance amount
+    INITIAL_PREISSUED_EXCEEDS_MAX_ISSUANCE = -11, // initial pre issued amount exceeds max issuance amount
+    //: Not allowed to use details with invalid json structure
     INVALID_CREATOR_DETAILS = -12,                        // details must be a valid json
+    //: Not allowed to set trailing digits count more than max trailing digits count (for now - 6)
```

Comment:
Not allowed to set trailing digits count greater than maximum trailing digits count (6 at the moment)